e2e/execution_test.go

```diff
@@ -315,50 +237,66 @@ func testExecution(t *testing.T) {
 			}
 		)
 		t.Run("create executions", func(t *testing.T) {
+			msgs := make([]sdk.Msg, 0)
+			for i := 0; i < n; i++ {
+				hash, err := hash.Random()
+				require.Nil(t, err)
+				msg := executionmodule.MsgCreate{
+					Signer:       engineAddress,
+					TaskKey:      taskKey,
+					EventHash:    hash,
+					ExecutorHash: executorHash,
+					Inputs:       inputs,
+					Price:        price,
+				}
+				msgs = append(msgs, msg)
+			}
+			execsHash := lcdBroadcastMsgs(msgs)
+			// split hash
```

Comment:
as there are multiple msgs in this tx, the results are concatted and need to be splitted.
as all results are hash with a fixed size, I can simply split the slice of byte into multiple slice of byte.